### DISCUSSION TITLE
能否在api中返回查詢經文所在的頁碼

### DISCUSSION THREAD

**Discussion by sunyrain** [DILA-edu/cbeta-api#45] [Q&A]
能否在api中返回查詢經文所在的頁碼？類似於《新集藏經音義隨函錄(第13卷-第30卷)》卷17：「[幻-ㄠ+糸]術」(CBETA, K35, no. 1257, p. 189b6), 組字式 escape 之後：[幻-ㄠ%2B糸] 里的 CBETA, K35, no. 1257, p. 189b6

**Comment by RayCHOU** [comment]
與 #44 重複。
